Scripts/script-TaniumAskQuestion.yml

```diff
@@ -90,6 +92,9 @@ script: |-
       if (table.Contents.length === 0) {
          return 'No Results';
       }
+      var title = 'name' in args ? args.name : "Tanium ask question";
```

Comment:
We usually use args.name and not name in args (this will return undefined if name doesn't exist in object, which is equivalent to false)